public/locales/en-US/send.ftl

```diff
@@ -117,3 +117,146 @@ passwordIsSet = Password set
 passwordSetError = This password could not be set
+#
+#
+#
+# shared ->
+#
+#
+#
+#  title = Firefox Send
+#  siteFeedback = Feedback
+#  importingFile = Importing…
+#  encryptingFile = Encrypting…
+#  decryptingFile = Decrypting…
+#  downloadCount = { $num ->
+#          [one] 1 download
+#         *[other] { $num } downloads
+#      }
+#  timespanHours = { $num ->
+#          [one] 1 hour
+#         *[other] { $num } hours
+#      }
+#  copiedUrl = Copied!
+#  unlockInputPlaceholder = Password
+#  unlockButtonLabel = Unlock
+#  downloadButtonLabel = Download
+#  downloadFinish = Download complete
+#  fileSizeProgress = ({ $partialSize } of { $totalSize })
+#  sendYourFilesLink = Try Firefox Send
+#  errorPageHeader = Something went wrong!
+#  fileTooBig = That file is too big to upload. It should be less than { $size }.
+#  linkExpiredAlt = Link expired
+#  notSupportedHeader = Your browser is not supported.
+#  notSupportedLink = Why is my browser not supported?
+#  notSupportedOutdatedDetail = Unfortunately this version of Firefox does not support the web technology that powers Firefox Send. You’ll need to update your browser.
+#  updateFirefox = Update Firefox
+#  deletePopupCancel = Cancel
+#  deleteButtonHover = Delete
+#  footerLinkLegal = Legal
+#  footerLinkPrivacy = Privacy
+#  footerLinkCookies = Cookies
+#  passwordTryAgain = Incorrect password. Try again.
+#  javascriptRequired = Firefox Send requires JavaScript
+#  whyJavascript = Why does Firefox Send require JavaScript?
+#  enableJavascript = Please enable JavaScript and try again.
+#  # A short representation of a countdown timer containing the number of hours and minutes remaining as digits, example "13h 47m"
+#  expiresHoursMinutes = { $hours }h { $minutes }m
+#  # A short representation of a countdown timer containing the number of minutes remaining as digits, example "56m"
+#  expiresMinutes = { $minutes }m
+#  # A short status message shown when the user enters a long password
+#  maxPasswordLength = Maximum password length: { $length }
+#  # A short status message shown when there was an error setting the password
+#  passwordSetError = This password could not be set
+#
+#
+#
+# vnext ->
+#
+#
+#
+introTitle = Simple, private file sharing
+introDescription = Firefox Send lets you share files with end-to-end encryption and a link that automatically expires. So you can keep what you share private and make sure your stuff doesn't stay online forever.
+notifyUploadEncryptDone = Your file is encrypted and ready to send
+archiveExpiryInfo = Expires after { $downloadCount } or { $timespan }
+timespanMinutes = { $num ->
+        [one] 1 minute
+       *[other] { $num } minutes
+    }
+timespanDays = { $num ->
+        [one] 1 day
+       *[other] { $num } days
+    }
+timespanWeeks = { $num ->
+        [one] 1 week
+       *[other] { $num } weeks
+    }
+fileCount = { $num ->
+    [one] 1 file
+   *[other] { $num } files
+}
+totalSize = Total size: { $size }
```

Comment:
What's $size? How is it formatted?

Reply:
I added a comment in my PR...this is total allowable size measured in terms of GB. this will return something like 2.5GB

Reply:
Right, the question is more about how we format that number. Do we have formatting by locale, do we just assume $number$unit, no spaces?

Reply:
https://github.com/mozilla/send/blob/master/app/utils.js#L98

Reply:
OK, that's not great, but we can look into it later. This would need at least one string to define how the string is formatted (e.g. space or not, and which kind, between number and unit). Normally we expose for localization also the units.